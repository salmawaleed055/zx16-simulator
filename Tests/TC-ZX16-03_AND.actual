Loaded 10 bytes into memory
0x0000: 2F71    xori x5, 23
0x0002: 7471    xori x1, 58
0x0004: 5EB1    xori x2, 47
0x0006: 8468    and x1, x2
0x0008: FFC7    Unknown opcode

--- Final State ---
x0: 0x0000
x1: 0x002a
x2: 0x002f
x3: 0x0000
x4: 0x0000
x5: 0x0017
a0: 0x0000
a1: 0x0000
PC: 0x0008
---------------------

Simulation finished.
