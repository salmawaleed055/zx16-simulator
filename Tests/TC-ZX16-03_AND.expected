0x0000: 7471    li x1, 58
0x0002: 5EB1    li x2, 47
0x0004: 8468    and x1, x2
0x0006: FFC7    ecall 0x3FF

--- Final State ---
x0: 0x0000
x1: 0x002A
x2: 0x002F
x3: 0x0000
x4: 0x0000
x5: 0x0000
a0: 0x0000
a1: 0x0000
PC: 0x0008
---------------------
Simulation finished.